modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -9,10 +9,20 @@ endif::[]
 
 :_mod-docs-content-type: CONCEPT
 [id='modifying-install-config-for-dual-stack-network_{context}']
-= Deploying with dual-stack networking
+= Deploying static IP addressing with dual-stack networking
+
+When deploying static IP addressing with dual-stack networking for the bootstrap VM, the bootstrap VM functions with a single IP version. Configuring dual-stack with a static IP set in install-config requires one virtual IP address (VIP) each for API and Ingress. Add secondary VIPs after deployment.
```

Comment:
This information should be in the note section too so it's clear it only applies to static IP deployments.

Reply:
Updated as suggested.